utils/gov/formatData.ts

```diff
@@ -0,0 +1,177 @@
+// export function formatDate(voting_end_time: string): string {
+//     const date = new Date(voting_end_time);
+    
+//     const day = String(date.getUTCDate()).padStart(2, '0');
+//     const month = String(date.getUTCMonth() + 1).padStart(2, '0');
+//     const year = String(date.getUTCFullYear()).slice(-2);
+
+//     return `${day}/${month}/${year}`;
+// }
+export function formatDate(voting_end_time: string): string {
+    const date = new Date(voting_end_time);
+
+    const day = date.getDate();
+    const month = date.toLocaleString('default', { month: 'short' });
+    const year = date.getFullYear() % 100;
+
+    const dayWithOrdinal = addOrdinalIndicator(day);
+
+    return `${dayWithOrdinal} ${month},${year}`;
+}
+
+// Function to add ordinal indicator (st, nd, rd, or th) to a number
+function addOrdinalIndicator(day: number): string {
```

Comment:
this function's only use is in another unsed function

Reply:
Deleted both